tensorflow_addons/text/tests/crf_test.py

```diff
@@ -559,3 +560,60 @@ def test_crf_decode_save_load(tmpdir):
             "seq_len": np.array([10]),
         }
     )
+
+
+@pytest.mark.parametrize(
+    "potentials,sequence_length",
+    [
+        # performs masking
+        pytest.param(
+            tf.random.normal([2, 12, 3]),
+            tf.constant([8, 10]),
+        ),
+        # does not perform masking
+        pytest.param(
+            tf.random.normal([4, 8, 10]),
+            tf.constant([8, 8, 8, 8]),
+        ),
+    ],
+)
+def test_crf_decode_forward_mask(potentials, sequence_length):
+    # mimics setup of the `_multi_seq_fn` closure in `crf_decode`
+    initial_state = tf.slice(potentials, [0, 0, 0], [-1, 1, -1])
+    initial_state = tf.squeeze(initial_state, axis=[1])
+    inputs = tf.slice(potentials, [0, 1, 0], [-1, -1, -1])
+
+    sequence_length_less_one = tf.maximum(
+        tf.constant(0, dtype=tf.int32), sequence_length - 1
+    )
+
+    n_tags = potentials.shape[-1]
+    transition_params = tf.random.normal([n_tags, n_tags])
+
+    backpointers, last_score = text.crf_decode_forward(
```

Comment:
Since the variable `last_score` is not used in the context. The standard way to handle it is to name this variable `_`.